numpy/core/_add_newdocs.py

```diff
@@ -2690,10 +2690,17 @@
 
     Notes
     -----
-    Starting in NumPy 1.9, astype method now returns an error if the string
-    dtype to cast to is not long enough in 'safe' casting mode to hold the max
-    value of integer/float array that is being casted. Previously the casting
-    was allowed even if the result was truncated.
+    Casting between a simple data type and a structured one is possible only
+    for "unsafe" casting.  Casting to multiple fields is allowed, but casting
+    from multiple fields is not.
+
+    .. versionchanged:: 1.18.0
```

Comment:
I think this might be clearer as:
```
.. versionchanged:: 1.18.0
   Casting between a simple data type and ...

.. versionchanged:: 1.9.0
   Casting rom numeric to ...

Reply:
Will do this once we have converged on an implementation...

Reply:
I changed `versionchanged` as recommended - I think this text at least is OK.